v1/examples/pipeline/stages/cd/helm.yaml

```diff
@@ -0,0 +1,25 @@
+version: 1
+kind: pipeline
+spec:
+  stages:
+    - type: deployment
+      failure:
+        - errors:
+            - all
+          action:
+            type: fail
+      spec:
+        service:
+          serviceRef: test
+        environment:
+          environmentRef: dev
+          deployToAll: false
+          infrastructureDefinitions:
+            - identifier: aks
+        deployment: native-helm
+        steps:
+          - type: helm_deploy
```

Comment:
shouldnt this be helm-deploy? as its enum

Reply:
You are right. I have corrected it.